documents/Libraries/stdlib/stdlib_osl_impl.mtlx

```diff
@@ -622,21 +622,6 @@
   <implementation name="IM_contrast_vector4_osl" nodedef="ND_contrast_vector4" file="stdlib/osl/mx_contrast_vector4.osl" function="mx_contrast_vector4" language="osl"/>
   <implementation name="IM_contrast_vector4FA_osl" nodedef="ND_contrast_vector4FA" file="stdlib/osl/mx_contrast_float_vector4.osl" function="mx_contrast_float_vector4" language="osl"/>
 
-  <!-- <range> Supplemental Node -->
```

Comment:
These are implemented by nodegraphs so these should be removed as it defines the implementations twice.
Also the OSL files listed for these implementations don't exist.